src/App.js

```diff
@@ -1,25 +1,36 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import countries from './data/countries.json';
+import Navbar from './components/Navbar';
+import { Route, Switch } from 'react-router-dom';
+
 import './App.css';
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
+        <Route>
+
+          <Switch>
+
+            <div className="row">
+
+              <div className="col-5"> Wiki Countries
```

Comment:
maybe you could build 2 components for this part and then pass them into the route